todoapp/app/src/mock/java/com/example/android/architecture/blueprints/todoapp/data/FakeTasksRemoteDataSource.java

```diff
@@ -52,44 +56,64 @@ public static FakeTasksRemoteDataSource getInstance() {
     @Override
     public Observable<List<Task>> getTasks() {
         Collection<Task> values = TASKS_SERVICE_DATA.values();
-        return Observable.from(values).toList();
+        return Observable.from(values).toList().repeatWhen(new Func1<Observable<? extends Void>, Observable<Boolean>>() {
```

Comment:
Isn't `Observable.from(values).toList()` == `Observable.just(values)` ? 
Also, can we use lambdas? 
and put `repeatWhen` on a different line.